ci/jenkins/jobs/macros.yaml

```diff
@@ -34,6 +34,24 @@
                   --vcuser "${{VC_USER}}" --datacenter "${{DATACENTERNAME}}" --datastore "${{DATA_STORE}}" \
                   --vcCluster "${{VC_CLUSTER}}" --resourcePool "${{RESOURCEPOOLPATH}}" --vcNetwork "${{VMC_NETWORK_1}}" \
                   --virtualMachine "${{VIRTUAL_MACHINE}}" --goVcPassword "${{GOVC_PASSWORD}}" --testType aks
+- builder:
+      name: deploy-vm-with-agent-and-run-eks-tests-for-nephe
+      builders:
+          - shell: |-
+                #!/bin/bash
+                ./ci/jenkins/nephe-ci.sh --buildnumber "${{BUILD_NUMBER}}" --vchost "${{VC_HOST}}" \
```

Comment:
follow https://github.com/antrea-io/nephe/pull/5, ${{BUILD_NUMBER}} should be ${BUILD_NUMBER}, so does other vars